test/util/getNvmData.test.ts

```diff
@@ -0,0 +1,42 @@
+import fetch from 'node-fetch';
+
+import { getLatestNvmVersion } from '../../util-node/getNvmData';
+
+jest.mock('node-fetch');
+
+const mockFetch = fetch as jest.MockedFunction<typeof fetch>;
+
+const prepareMockFetch = ({
+  response = [{ name: 'mockVersionString' }] as any,
```

Comment:
`as NvmTagsResponse`?

Reply:
Nah, this is specifically for supporting being able to pass in invalid mock responses. We need to break type checking to confirm that the type guards work correctly.